src/app/questions/file/file-display/file-display.component.ts

```diff
@@ -14,7 +14,9 @@ export class FileDisplayComponent implements OnInit, OnChanges {
 
   @Input() fileType = 'any';
   @Input() file: any;
+  @Input() isFileComponent?: boolean;
```

Comment:
file display component also using in chat but in chat we didn't want to show file name, file type icon and option to remove file. so because of that i added this optional input. this only pass from file component as true but this will not pass from chat.